lms/static/scripts/frontend_apps/components/dashboard/OrganizationActivity.tsx

```diff
@@ -34,6 +35,8 @@ export default function OrganizationActivity() {
     organizationPublicId: string;
   }>();
 
+  usePageTitle('All courses - Hypothesis');
```

Comment:
Addition of the `- Hypothesis` suffix here could be moved into the hook, since I expect we'll want this suffix for most or all pages. If there are a minority where we don't need it, the hook could have an option to turn it off.